content/applications/inventory_and_mrp/manufacturing/maintenance/add_new_equipment.rst

```diff
@@ -0,0 +1,63 @@
+=================
+Add new equipment
+=================
+Using Odoo Maintenance, it is possible to track individual pieces of equipment, along with
+information about their maintenance requirements. To add a new piece of equipment, navigate to the
+:guilabel:`Maintenance` module, select :menuselection:`Equipments --> Machines & Tools --> Create`,
+and configure the equipment as follows:
+
+- :guilabel:`Equipment Name`: the product name of the piece of equipment
+- :guilabel:`Equipment Category`: the category that the equipment belongs to
+- :guilabel:`Company`: the company that owns the equipment
+- :guilabel:`Used By`: Specify if the equipment is used by a specific employee, department, or both.
```

Comment:
lowercase to match formatting in the rest of the list.

also, examples/use cases here?
when's it better to assign a person vs a department? how do you create/edit a department?
```suggestion
- :guilabel:`Used By`: specify if the equipment is used by a specific employee, department, or both.
```